From 5fa711f46837116fd1b9f848b0365a0001b40fb4 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Wed, 19 Nov 2014 02:38:44 +0100
Subject: [PATCH] tuna hdpi build

---
 device.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/device.mk b/device.mk
index 3501de9..e8830d0 100644
--- a/device.mk
+++ b/device.mk
@@ -31,8 +31,8 @@ DEVICE_PACKAGE_OVERLAYS := device/samsung/tuna/overlay
 # currently contain all of the bitmaps at xhdpi density so
 # we do this little trick to fall back to the hdpi version
 # if the xhdpi doesn't exist.
-PRODUCT_AAPT_CONFIG := normal hdpi xhdpi
-PRODUCT_AAPT_PREF_CONFIG := xhdpi
+PRODUCT_AAPT_CONFIG := normal hdpi
+PRODUCT_AAPT_PREF_CONFIG := hdpi
 
 PRODUCT_PACKAGES := \
 	lights.tuna \
@@ -180,7 +180,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 	ro.opengles.version=131072
 
 PRODUCT_PROPERTY_OVERRIDES += \
-	ro.sf.lcd_density=320
+	ro.sf.lcd_density=240
 
 PRODUCT_PROPERTY_OVERRIDES += \
 	ro.hwui.disable_scissor_opt=true
-- 
2.1.3

