From 6a6d7af6788ab55bdc211830e6b157a58ee306e2 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Fri, 6 Mar 2015 16:42:44 +0100
Subject: [PATCH] bacon xhdpi build

---
 bacon.mk    | 4 ++--
 system.prop | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/bacon.mk b/bacon.mk
index bd727ca..b37bc90 100644
--- a/bacon.mk
+++ b/bacon.mk
@@ -61,8 +61,8 @@ PRODUCT_PROPERTY_OVERRIDES += \
 PRODUCT_PACKAGES += camera.bacon
 
 # Device uses high-density artwork where available
-PRODUCT_AAPT_CONFIG := normal hdpi xhdpi xxhdpi
-PRODUCT_AAPT_PREF_CONFIG := xxhdpi
+PRODUCT_AAPT_CONFIG := normal hdpi xhdpi
+PRODUCT_AAPT_PREF_CONFIG := xhdpi
 
 # call dalvik heap config
 $(call inherit-product-if-exists, frameworks/native/build/phone-xxhdpi-2048-dalvik-heap.mk)
diff --git a/system.prop b/system.prop
index e627885..8406730 100644
--- a/system.prop
+++ b/system.prop
@@ -2,7 +2,7 @@
 # system.prop for one+
 #
 
-ro.sf.lcd_density=480
+ro.sf.lcd_density=320
 rild.libpath=/system/vendor/lib/libril-qc-qmi-1.so
 persist.radio.add_power_save=1
 persist.radio.apm_sim_not_pwdn=1
-- 
2.3.1

