From 4daadab50d0d18c3853dae1f7f40f80c17928359 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Wed, 16 Mar 2016 00:42:55 +0100
Subject: [PATCH] hammerhead xhdpi build

---
 device.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/device.mk b/device.mk
index e701100..d1d0f0e 100644
--- a/device.mk
+++ b/device.mk
@@ -104,7 +104,7 @@ PRODUCT_COPY_FILES += \
     device/lge/hammerhead/spn-conf.xml:system/etc/spn-conf.xml
 
 PRODUCT_AAPT_CONFIG := normal
-PRODUCT_AAPT_PREF_CONFIG := xxhdpi
+PRODUCT_AAPT_PREF_CONFIG := xhdpi
 
 PRODUCT_CHARACTERISTICS := nosdcard
 
@@ -227,7 +227,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
     ro.opengles.version=196608
 
 PRODUCT_PROPERTY_OVERRIDES += \
-    ro.sf.lcd_density=480
+    ro.sf.lcd_density=320
 
 PRODUCT_PROPERTY_OVERRIDES += \
     persist.hwc.mdpcomp.enable=true
-- 
2.7.2

