From be0ea88a42e65f20b377ba8a4687d32ba7baf2b2 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Fri, 1 May 2015 17:59:58 +0200
Subject: [PATCH] hammerhead xhdpi build

---
 device.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/device.mk b/device.mk
index 2bc9590..92cd2d5 100644
--- a/device.mk
+++ b/device.mk
@@ -101,7 +101,7 @@ PRODUCT_COPY_FILES += \
     device/lge/hammerhead/spn-conf.xml:system/etc/spn-conf.xml
 
 PRODUCT_AAPT_CONFIG := normal
-PRODUCT_AAPT_PREF_CONFIG := xxhdpi
+PRODUCT_AAPT_PREF_CONFIG := xhdpi
 
 PRODUCT_CHARACTERISTICS := nosdcard
 
@@ -222,7 +222,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
     ro.opengles.version=196608
 
 PRODUCT_PROPERTY_OVERRIDES += \
-    ro.sf.lcd_density=480
+    ro.sf.lcd_density=320
 
 PRODUCT_PROPERTY_OVERRIDES += \
     persist.hwc.mdpcomp.enable=true
-- 
2.3.7

