From 2a2032ce9deef9313da1d5eb15dc993c83e66ef5 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Wed, 19 Nov 2014 02:13:29 +0100
Subject: [PATCH] hammerhead xhdpi build

---
 device.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/device.mk b/device.mk
index 794a11b..dd90b38 100644
--- a/device.mk
+++ b/device.mk
@@ -105,8 +105,8 @@ PRODUCT_TAGS += dalvik.gc.type-precise
 # currently contain all of the bitmaps at xhdpi density so
 # we do this little trick to fall back to the hdpi version
 # if the xhdpi doesn't exist.
-PRODUCT_AAPT_CONFIG := normal hdpi xhdpi xxhdpi
-PRODUCT_AAPT_PREF_CONFIG := xxhdpi
+PRODUCT_AAPT_CONFIG := normal hdpi xhdpi
+PRODUCT_AAPT_PREF_CONFIG := xhdpi
 
 PRODUCT_CHARACTERISTICS := nosdcard
 
@@ -227,7 +227,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
     ro.opengles.version=196608
 
 PRODUCT_PROPERTY_OVERRIDES += \
-    ro.sf.lcd_density=480
+    ro.sf.lcd_density=320
 
 PRODUCT_PROPERTY_OVERRIDES += \
     persist.hwc.mdpcomp.enable=true
-- 
2.1.3

