From e16e7bd97eabec67c9b1a0746f13a9b277636264 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Wed, 16 Mar 2016 00:43:32 +0100
Subject: [PATCH] mako hdpi build

---
 device.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/device.mk b/device.mk
index abaf1de..6de8bd3 100644
--- a/device.mk
+++ b/device.mk
@@ -21,7 +21,7 @@
 
 DEVICE_PACKAGE_OVERLAYS := device/lge/mako/overlay
 
-PRODUCT_AAPT_PREF_CONFIG := xhdpi
+PRODUCT_AAPT_PREF_CONFIG := hdpi
 
 PRODUCT_PACKAGES := \
     lights.mako \
@@ -117,7 +117,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 	ro.opengles.version=196608
 
 PRODUCT_PROPERTY_OVERRIDES += \
-	ro.sf.lcd_density=320
+	ro.sf.lcd_density=240
 
 # Audio Configuration
 # FIXME: Remove persist.audio.handset.mic and persist.audio.fluence.mode
-- 
2.7.2

