From 6720ae1731c0d08416fe1f80fe8978155721d9f6 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Thu, 5 Jan 2017 23:31:14 +0100
Subject: [PATCH] remove deprecated RomManager

---
 products/common.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/products/common.mk b/products/common.mk
index 17e0b2c..0ffba05 100644
--- a/products/common.mk
+++ b/products/common.mk
@@ -86,8 +86,8 @@ PRODUCT_COPY_FILES += \
 PRODUCT_COPY_FILES += \
 #    vendor/cyanogen/prebuilt/common/etc/init.d/10apps2sd:system/etc/init.d/10apps2sd
 
-PRODUCT_COPY_FILES +=  \
-    vendor/cyanogen/proprietary/RomManager.apk:system/app/RomManager.apk \
+#PRODUCT_COPY_FILES +=  \
+#    vendor/cyanogen/proprietary/RomManager.apk:system/app/RomManager.apk \
 
 # Enable SIP+VoIP on all targets
 PRODUCT_COPY_FILES += \
-- 
2.11.0

