From ac88d0ac5fa17449af7b84dbffc34f955b4f3f19 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Mon, 7 Jul 2014 15:52:02 +0200
Subject: [PATCH] tuna hdpi build

---
 device.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/device.mk b/device.mk
index 8d1c1bc..b5d2b37 100644
--- a/device.mk
+++ b/device.mk
@@ -25,8 +25,8 @@ DEVICE_PACKAGE_OVERLAYS := device/samsung/tuna/overlay
 # currently contain all of the bitmaps at xhdpi density so
 # we do this little trick to fall back to the hdpi version
 # if the xhdpi doesn't exist.
-PRODUCT_AAPT_CONFIG := normal hdpi xhdpi
-PRODUCT_AAPT_PREF_CONFIG := xhdpi
+PRODUCT_AAPT_CONFIG := normal hdpi
+PRODUCT_AAPT_PREF_CONFIG := hdpi
 
 PRODUCT_PACKAGES := \
 	lights.tuna \
@@ -178,7 +178,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 	ro.opengles.version=131072
 
 PRODUCT_PROPERTY_OVERRIDES += \
-	ro.sf.lcd_density=320
+	ro.sf.lcd_density=240
 
 PRODUCT_PROPERTY_OVERRIDES += \
 	ro.hwui.disable_scissor_opt=true
-- 
1.9.1

